README.md

```diff
@@ -64,6 +64,8 @@ Give relies on several npm commands to get you started:
 * `npm run dev` - Runs a one time build for development. No production files are created.
 * `npm run production` - Builds the minified production files for release.
 
+##### Note only compatible with Node 10
```

Comment:
I am going to remove this note after merging because it works fine on my node 11+ mac